Gemfile

```diff
@@ -99,6 +101,11 @@ gem "websocket-driver",               "~>0.6.3"
 gem "foreman_api_client",             ">=0.1.0",   :require => false, :git => "https://github.com/ManageIQ/foreman_api_client.git", :branch => "master"
 gem "ruport",                         "=1.7.0",                       :git => "https://github.com/ManageIQ/ruport.git", :tag => "v1.7.0-3"
 
+gem "uuid"
```

Comment:
If I'm reading the `Gemfile.lock` file this is probably only used by `xclarity_client` gem.  This could probably move to `manageiq-providers-lenovo` along with the `xclarity_client` gem below.